src/sass/_instance_detail_page.scss

```diff
@@ -1,10 +1,62 @@
 .instance-detail-page {
+  .p-panel__title {
+    display: flex;
+    flex-direction: column;
+
+    @include large {
+      align-items: center;
+      flex-direction: row;
+    }
+
+    .p-breadcrumbs {
+      margin-bottom: 0;
+      width: inherit;
+
+      .p-breadcrumbs__items {
+        margin-bottom: 0;
+      }
+
+      .p-breadcrumbs__item {
+        font-size: #{map.get($font-sizes, h4-mobile)}rem;
+        line-height: 2rem;
+        margin-bottom: 0;
+        margin-top: 0;
+        padding-top: 0;
+      }
+
+      .p-breadcrumbs__item:first-of-type {
+        margin-right: $sp-x-small;
+      }
+    }
+
+    .instance-name {
+      margin-right: $sph--x-small;
+      max-width: 35vw;
+      overflow: clip !important;
+    }
+
+    .actions-list {
+      line-height: 2rem;
```

Comment:
Why the line-height?

Reply:
This is a leftover of a previous change, before we introduced the new line on small and medium sizes. Because the `line-height` of the `p-panel__title` gets reduced from `2rem` to `1.5rem` below the large breakpoint, and this was causing a mis-alignment with the breadcrumbs. Now, instead, when we go below the large breakpoint, the breadcrumbs will always be on the first line and the status + actions on a new line, so we don't need to "force" the `line-height` of the latter to `2rem`.